### PR TITLE
react-fixed-dataTables replaced with react-table for support reasons.

### PR DESCRIPTION
**Pull request by Ab-hay**
*Issue # 13:*

Removed fixed-data tables and replaced it with react-table.

### CODE REVIEW

File: styles/style.scss [hunk 1]
```diff
@@ -4,7 +4,14 @@
 @import '~font-awesome/css/font-awesome.css';
 @import '~react-select/scss/default.scss';
 @import '~react-bootstrap-sweetalert/lib/css/animations.css';
-@import '~fixed-data-table/dist/fixed-data-table.min.css';
+@import '~react-table/react-table.css';
+
+body {
```
Comment (jpeddicord) [inline styles/style.scss]: Drop this part please. We can tweak the table styling later, but I'd rather not change the look & feel of the whole site.

File: browser/components/ContributionsTable.tsx [hunk 2]
```diff
@@ -97,84 +93,50 @@ class ContributionsTable extends React.Component<Props, State> {
...
-            }
-            width={100}
-          />
-          {tableProps.contributor ? (
-            ContributionsTable.contributorEntry(item, tableProps.contributor)
-          ) : (
-            <div />
-          )}
-        </Table>
+      <div key={count++}>
```
Comment (jpeddicord) [inline browser/components/ContributionsTable.tsx]: This `count` will always be `1` in this function. Having the key be `name` should be fine; did that not work?

File: browser/components/TableApprovalCell.tsx [hunk 3]
```diff
@@ -11,35 +11,27 @@
...
-      <Cell {...props}>
-        <div className="row">
-          <div className="col-md-6">
-            <Link className="btn btn-info" to={`/approvals/${id}`}>
-              Approve/Deny
-            </Link>
-          </div>
+      <div className="row center">
+        <div className="col-md-6">
+          <Link className="btn-info" to={`/approvals/${this.props.id}`}>
```
Comment (jpeddicord) [inline browser/components/TableApprovalCell.tsx:30]: This ends up looking strange in the table. Don't worry about fixing it now, I can help clean that up later.

**Review by jpeddicord — CHANGES_REQUESTED** [review]
This functions pretty dang well. Thanks for making this change! I like that it removes a bunch of code that's handled by react-table now.

I have a couple requests in the comments below.

File: browser/components/ContributionsTable.tsx (hunk 2 above)
Comment (Ab-hay) [reply browser/components/ContributionsTable.tsx]: yep, It should have worked fine. I have no idea how it got in there but i will fix it.

File: styles/style.scss (hunk 1 above)
Comment (Ab-hay) [reply styles/style.scss]: I will drop the body css.

**Review by jpeddicord — APPROVED** [review]

**Pull request merged**